### DISCUSSION TITLE
CORS Issue

### DISCUSSION THREAD

**Discussion by alexanderstahl93** [medusajs/medusa#5191] [Q&A]
Hi guys, i deployed the backend on heroku, and the storefront and admin panel on vercel. 

The storefront: https://opusone-store.vercel.app/, and admin panel: https://opusone-admin.vercel.app/login are accesible, **but it doesn't fetch any data from the database, because of a CORS issue** I am trying to fix since a few days. 

I don't know what to do, I did everything according to the documentation, and tried to fix it otherwise trough CORS middleware in the index.js, but it didnt work. 

**Can anyone please help me figure this out?**

![image](https://github.com/medusajs/medusa/assets/90091045/36a8bcb2-ef85-4f7b-baf2-dcc136430d6c)

![image](https://github.com/medusajs/medusa/assets/90091045/83d19fb4-f4ac-450f-a511-b31b590a8634)

**Comment by pevey** [comment]
What are your CORS settings on your backend on Heroku?

https://docs.medusajs.com/troubleshooting/cors-issues

**Comment by SianAliMalik2418** [comment]
Were you able to solve it @alexanderstahl93 ?
